docs/en/observability/aws-elastic-serverless-forwarder.asciidoc

```diff
@@ -0,0 +1,99 @@
+:aws: AWS
+
+[[aws-elastic-serverless-forwarder]]
+= Elastic Serverless Forwarder
+
+// should we expand titleabbrev to AWS or Amazon Web Services Elastic Serverless Forwarder? What is correct nomenclature?
+
+++++
+<titleabbrev>AWS Elastic Serverless Forwarder</titleabbrev>
+++++
+
+:keywords: serverless
+:description: The Elastic Serverless Forwarder is an Amazon Web Services ({aws}) Lambda function that ships logs from your {aws} environment to Elastic.
+
+The Elastic Serverless Forwarder is an Amazon Web Services ({aws}) Lambda function that ships logs from your {aws} environment to Elastic.
+
+[discrete]
+[[aws-serverless-forwarder-overview]]
+= Overview
+The Elastic Serverless Forwarder can forward {aws} data to self-managed or cloud-hosted Elastic environments. It supports the following inputs:
+
+- Amazon S3 SQS (Simple Queue Service) event notifications
+- Amazon Kinesis Data Streams
+- Amazon CloudWatch Logs subscription filters
+- Direct Amazon SQS message payload
+
+[role="screenshot"]
+image::aws-serverless-lambda-flow.png[AWS Lambda flow]
+
+The Lambda deployment sets up an SQS queue automatically. This is used to trigger a new function invocation that Lambda uses to continue from exactly where the last function run was terminated. By default a Lambda function runs for a maximum of 15 minutes. When processing event data there’s a possibility that the function may be exited by {aws} in the middle of processing. The function handles this scenario gracefully by keeping track of the last offset processed.
```

Comment:
The Lambda deployment sets up a continuing SQS queue automatically.

Reply:
Think I edited that original reference out lol - what do you mean by "continuing"?
I don't think Amazon refer to a "continuing SQS queue" in their docs so trying to understand what this means in this context e.g. "persistent"?

Reply:
Yes, its an SQS queue but we call it continuing queue (our name). This is what we are showing in our diagram as well.
This helps us to differentiate and refer to the specific queue (as you can see there are many) by its purpose (helps with checkpointing and continue where where we left of in the last run).

Reply:
Ah, thanks - I see it in the diagram now. Updating locally, will push another commit and then ask for a peer review from an Obs writer.